net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,96 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all"
+
+libVersion="$portVersion"
+
+PROVIDES="
+	gwenhywfar = $portVersion
+	cmd:gct_tool = $portVersion
+	cmd:gsa = $portVersion
+	cmd:gwbuild = $portVersion
+	cmd:gwenhywfar_config = $portVersion
+	cmd:mklistdoc = $portVersion
+	cmd:typemaker = $portVersion
+	cmd:typemaker2 = $portVersion
+	cmd:xmlmerge = $portVersion
+	lib:libgwenhywfar = $libVersion
+	lib:libgwengui_cpp = $libVersion
+	lib:libgwengui_gtk3 = $libVersion
```

Comment:
What is the SONAME of these libraries? Would it make sense to use `$libVersionCompat` here?